README.md

```diff
@@ -25,26 +24,84 @@ Welcome to our source speech repository about [mobx-state-tree](https://github.c
 ### Model
 > This is about creating our first model
 
-#### [#12](https://github.com/Dedetat/powerpuff-yourself/pull/12/files) - Create the first model and instanciate it
-> Write our first model of **Powerpuff**, we can call it `Powerpuff` (damn, clever)
- 1. We import `types` from `mobx-state-tree`, this is the most common used API
- 2. `types` is used to ... type fields 👌, we can already use it to describe models ! And  `types.model` is the way to go !
+#### [#30](https://github.com/Dedetat/powerpuff-yourself/pull/30/files) - Create the first model and instanciate it
+> Writing our first model of **Powerpuff** is pretty simple. We'll call it `Powerpuff` (damn, clever 🤓)
+ 1. Import `types` from `mobx-state-tree`, this is the most common used API
+ 2. `types` is used to ... type fields 👌, we can also use it to describe models ! And  `types.model` is the way to go !
  3. Here we create a model with only one field, a `name` which is a string !
  4. Ok, let's instanciate our model with `Powerpuff.create({})`
     - `Powerpuff` is our model
     - `create` means we instanciate a new tree, a new instance
     - `{}` is the snapshot, the raw data, to use to instanciate a new powerpuff !
  5. Let's print it with a `console.log` !
-    - You can see we use `powerpuff.toJSON()` to convert our tree to a snapshot.
- 6. 🤔 oups, it doesn't work at all ! This is because, by default, a field is required.
- 7. Let's instanciate our powerpuff, it will be... hm... `Rebelle` (this is Blossom in french)
+    - As you can see, we use `powerpuff.toJSON()` to convert our tree to a snapshot
+ 6. 🤔 oups, it doesn't work at all ! This is because, by default, a field is required
+ 7. Let's instanciate our powerpuff, it will be... hm... `Rebelle` (the french name of Buttercup)
  8. Print it
  9. It works 🎉
 
+#### [#31](https://github.com/Dedetat/powerpuff-yourself/pull/31/files) - Avoid required fields
```

Comment:
When we say "Avoid required fields", it feels that mobx-state-tree is a pain to use because fields are required.
I have no idea right now, but I think we should change that a more positive way

Reply:
`Make fields optional` ?

Reply:
Yes ok